hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -318,11 +337,13 @@ private static void modify(FileSystem fs, Path metadataFolder, Properties modify
         modifyFn.accept(props, modifyProps);
         if (props.containsKey(TABLE_CHECKSUM.key()) && validateChecksum(props)) {
```

Comment:
Let's extract this whole conditional as a `persist` method (for ex), since it's pretty much replicated in 3 places

Reply:
Done.